WordPressKitTests/WordPressComRestApiTests+AsyncAwait.swift

```diff
@@ -0,0 +1,94 @@
+import XCTest
+import OHHTTPStubs
+import WordPressShared
+
+@testable import WordPressKit
+
+@available(iOS 15.0.0, *)
+class WordPressComRestApiAsyncAwaitTests: XCTestCase {
+
+    private let url = "https://public-api.wordpress.com/rest/v1.1/sites/0/media/"
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        HTTPStubs.removeAllStubs()
+    }
+
+    func testSuccessfullGetCall() async throws {
+        stub(condition: isAbsoluteURLString(url)) { _ in
+            let stubPath = OHPathForFile("WordPressComRestApiMedia.json", type(of: self))
+            return fixture(filePath: stubPath!, headers: ["Content-Type" as NSObject: "application/json" as AnyObject])
+        }
+
+        let api = WordPressComRestApi(oAuthToken: "fakeToken")
+        let (data, response) = try await api.get(url)
+
+        XCTAssertNotNil(data)
+        XCTAssertEqual(response.mimeType, "application/json")
+    }
+
+    func testBadServerResponseGetCall() async throws {
+        stub(condition: isAbsoluteURLString(url)) { _ in
+            let notConnectedError = NSError(domain: NSURLErrorDomain, code: URLError.badServerResponse.rawValue)
+            return HTTPStubsResponse(error: notConnectedError)
+        }
+
+        do {
+            let api = WordPressComRestApi(oAuthToken: "fakeToken")
+            let (_, _) = try await api.get(url)
```

Comment:
Nitpick. Since this is ignored, we can streamline to:
```suggestion
            _ = try await api.get(url)
```